android/forms-screenlet-demo/app/src/main/res/layout/layout_login.xml

```diff
@@ -84,6 +84,29 @@
 
 		</RelativeLayout>
 
+		<RelativeLayout
```

Comment:
What do you think about we use a divider here or maybe a text with something like "New user? Sign up"?